etc/ATSCam/primary_hdu.header

```diff
@@ -10,15 +10,15 @@ DATE-OBS=                      / Date of the observation (image acquisition)
 DATE-END=                      / end date of the observation
 MJD     =                      / Modified Julian Date that the file was written
 MJD-OBS =                      / Modified Julian Date of observation
```

Comment:
There is meant to be an MJD-END. Have you been following the changes in LSE-400 and the associated LCR?

Reply:
I think @ktlim  wants TIMESYS to be present and set to TAI.

Reply:
Once we have settle the source (CSC/Event/payload), please add them to the mapping document here: https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=SYSENG&title=Auxiliary+Telescope+Header+Information+Topic+Mapping